src/stories/components/reset-button/reset-button.tsx

```diff
@@ -6,27 +6,44 @@ import { Close } from '../svg/close';
 interface Props {
   onClick?: () => void;
   disabled?: boolean;
+  label?: string;
+  isWithIcon?: boolean;
+  labelMobile?: string;
 }
 
-export default (props: Props) => {
+export default ({ onClick, disabled, label = 'Reset Filters', isWithIcon = true, labelMobile }: Props) => {
```

Comment:
we can use `hasIcon` here as it is less verbose and still refers to a boolean according to the conventions

Reply:
ok, I will make the change